pkg/es/config/config.go

```diff
@@ -114,6 +147,41 @@ type TagsAsFields struct {
 	Include string `mapstructure:"include"`
 }
 
+// Sniffing sets the sniffing configuration for the ElasticSearch client, which is the process
+// of finding all the nodes of your cluster. Read more about sniffing at
+// https://github.com/olivere/elastic/wiki/Sniffing.
+type Sniffing struct {
+	// Enabled, if set to true, enables sniffing for the ElasticSearch client.
+	Enabled bool `mapstructure:"enabled"`
+	// UseHTTPS, if set to true, sets the HTTP scheme to HTTPS when performing sniffing.
+	UseHTTPS bool `mapstructure:"use_https"`
+}
+
+type BulkProcessing struct {
+	// MaxBytes, contains the number of bytes which specifies when to flush.
+	MaxBytes int `mapstructure:"max_bytes"`
+	// FlushInterval is the interval at the end of which a flush occurs.
+	FlushInterval time.Duration `mapstructure:"flush_interval"`
+	// Workers contains the number of concurrent workers allowed to be executed.
+	Workers int `mapstructure:"workers"`
+	// MaxActions contain the number of added actions which specifies when to flush.
+	MaxActions int `mapstructure:"max_actions"`
```

Comment:
please place related fields next to each other:
  1. MaxBytes
  2. MaxActions
  3. FlushInterval
  4. Workers